nservicebus/pipeline/unit-of-work.md

```diff
@@ -27,6 +27,7 @@ The `End()` method is called when the processing is complete. If there is an exc
 
 This gives a way to perform different actions depending on the outcome of the message(s).
 
+partial: imanageunitsofwork-outbox
```

Comment:
The idea was to include this note also in upgrade guide but I'm not sure what is the best place.
The best place I could come up with was http://docs.particular.net/nservicebus/upgrades/5to6/messaging but that still feels off.